src/pycram/utils.py

```diff
@@ -157,6 +159,105 @@ def __exit__(self, *_):
             os.close(fd)
 
 
+def adjust_camera_pose_based_on_target(cam_pose: Pose, target_pose: Pose,
+                                       camera_description: CameraDescription) -> Pose:
+    """
+    Adjust the camera pose based on the target pose.
```

Comment:
That does not explain what this method does

Reply:
add a clearer and more detailed explanation.